.circleci/config.yml

```diff
@@ -298,18 +299,43 @@ jobs:
       - run:
           name: Validate Swagger definitions
           working_directory: ~/workbench/api
-          command: ./project.rb validate-swagger --project-prop verboseTestLogging=yes
+          command: |
+            if [ "$CIRCLE_NODE_INDEX" == 0 ]; then
+              ./project.rb validate-swagger --project-prop verboseTestLogging=yes
```

Comment:
FYI, if you're testing this, note that this command isn't doing anything right now. https://precisionmedicineinitiative.atlassian.net/browse/RW-5073. No change is needed here, though.

Reply:
good to know.